com.unity.render-pipelines.high-definition/Runtime/Material/Lit/LitTessellation.shader

```diff
@@ -410,9 +410,9 @@ Shader "HDRP/LitTessellation"
             #define SCENEPICKINGPASS
             #include "Packages/com.unity.render-pipelines.high-definition/Runtime/Material/Material.hlsl"
             #include "Packages/com.unity.render-pipelines.high-definition/Runtime/Material/Lit/Lit.hlsl"
+            #include "Packages/com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/PickingSpaceTransforms.hlsl"
```

Comment:
I had to move this include here, since the tesselation algorithms are located inside LitData.hlsl.

Reply:
yes it make sense. 
So just to be coherent, we should move it in all shaders (axf.sahder, litTessellation.sahder, Lit.shader, LayeredLit.shader, LayeredLitTessellation.shader) and we also need to change it for shader graph.

this is located in HDSahderPasses.cs

here

            IncludeCollection GenerateIncludes()
            {
                var includes = new IncludeCollection();

                includes.Add(CoreIncludes.CorePregraph);
                includes.Add(CoreIncludes.kPassPlaceholder, IncludeLocation.Pregraph);
                includes.Add(CoreIncludes.CoreUtility);
                includes.Add(CoreIncludes.kShaderGraphFunctions, IncludeLocation.Pregraph);
                includes.Add(CoreIncludes.kPickingSpaceTransforms, IncludeLocation.Pregraph);
                includes.Add(CoreIncludes.kPassDepthOnly, IncludeLocation.Postgraph);

                return includes;
            }
            
            move  includes.Add(CoreIncludes.kPickingSpaceTransforms, IncludeLocation.Pregraph); below includes.Add(CoreIncludes.CorePregraph);

Reply:
Good call.

Reply:
Done.